src/traces/isosurface/calc.js

```diff
@@ -0,0 +1,32 @@
+/**
+* Copyright 2012-2019, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+var colorscaleCalc = require('../../components/colorscale/calc');
+
+module.exports = function calc(gd, trace) {
+
+    var vMin = trace.isomin;
+    var vMax = trace.isomax;
+    if(vMin === undefined) vMin = Math.min.apply(null, trace.value);
+    if(vMax === undefined) vMax = Math.max.apply(null, trace.value);
+
+    if(vMin === vMax) return;
```

Comment:
This case is already covered in `calcColorscale`:

https://github.com/plotly/plotly.js/blob/39915526327c657055feebc5e13b762616d52ebd/src/components/colorscale/calc.js#L39-L42